### ISSUE TITLE
upgraded to 2.3.1 now the world is tesselated and can't log in

### ISSUE THREAD

**Issue by kreezxil** [Nuchaz/BiblioCraft#463]
https://pastebin.com/Tx9Sq7Ah

**Comment by Nuchaz** [comment]
Related to #455 and #461 
I am still trying to figure this out. I haven't been able to reproduce yet on my computer. Maybe since your world is crashing when you try to log on, could you upload a copy of it and share me a link? If I could test with a world that won't even log in, that might help me figure this out. If you are using a modpack, a link to that would be helpful too so I don't have to download all the individual mods.

**Comment by kreezxil** [comment]
I see you say "on my computer", this is a dedicated server on the internet, I believe there's a difference between the dedis and internal servers.

anyhow: https://kreezcraft.com/special/world4u.zip

And the pack is Colonial Expansion and can be found on the Twith App (previously Curse Launcher). 
https://minecraft.curseforge.com/projects/colonial-expansion

For each version I have the cursepack, multimc version of that pack, and a server pack.

**Comment by kreezxil** [comment]
so i removed the region that was specified and got a new bug report exactly what I've submitted.

However when I started the world again and logged in i got the following in console, it may help you track things. Note, there are no bookcases in the chunk nor neighboring chunks that I am in.

```
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:46] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
[10:50:47] [Server thread/WARN]: Skipping BlockEntity with id minecraft:bookcase
```

**Comment by Nuchaz** [comment]
Thanks, I'll give it a try and have a look.
When I say on my computer I just mean in my attempts to to reproduce on my equipment. I've tried both SSP game and SMP games with a dedicated server. This seems to be a client side issue though, so it will likely happen in either.

**Comment by kreezxil** [comment]
I was thiinking since you were unable to replicate it. It might be something brought on by adding your mod to an already existing world vs one where the world is generated from the beginning with your mod in it.

**Comment by Nuchaz** [comment]
This is reliably crashing on me trying to load the world so with some experimentation I was about to narrow down and figure out what element the crash is occuring on. I am still working on figuring out a fix, but I am much closer now that I know what specifically is crashing.

**Comment by Nuchaz** [comment]
I have a fix out for this. I rolled the fix into BiblioCraft v2.1.4 for MC 1.9.4, BiblioCraft v2.2.4 for MC 1.10.2 and BiblioCraft v2.3.2 for MC 1.11.2. Closing the issue. You should be able to load worlds that have been permanently crashed by this bug.  Let me know if you experience anything weird.
